docker/build_matrix.yaml

```diff
@@ -36,46 +36,46 @@
 - BASE_IMAGE: nvidia/cuda:11.3.1-cudnn8-devel-ubuntu20.04
   CUDA_VERSION: 11.3.1
   PYTHON_VERSION: '3.9'
-  PYTORCH_VERSION: 1.11.0
+  PYTORCH_VERSION: 1.12.0
   TAGS:
-  - mosaicml/pytorch:1.11.0_cu113-python3.9-ubuntu20.04
+  - mosaicml/pytorch:1.12.0_cu113-python3.9-ubuntu20.04
   - mosaicml/pytorch:latest
   TARGET: pytorch_stage
   TORCHVISION_VERSION: 0.12.0
```

Comment:
I think the torchvision version will need an update -- they usually release a new torchvision version for each pytorch version.